content/blog/2023/2022blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2022 Progress!"
+author: "ghostway"
+published: 2023-01-14
+---
+
+2022 has been a great year for Leela. A lot of new contributors appeared and made significant improvements; overall, Leela has become considerably stronger and more interesting.
+This year has brought huge changes to how the search was conducted, network architecture, available backends and more.
+<!--more-->
+
+I'll dive into some of them.
+
+#### Network Architecture
+In the AlphaZero paper, Google used Resnets to predict the value and policies of the position. In 2022, *Arcturai* after a lot of work
+presented attention-based policy. Attention is a well known block that was already used in Imagenet (for more details about attention, see [Attention Is All You Need](https://arxiv.org/abs/1706.03762)). A blog on that will come soon™!
+That opened the path to replacing the whole network with a transformer-like architecture. With *danielmonoroe*'s work and improvements -- Smolgen being the major
+improvement so far -- BT2 is currently the strongest network in existence (for details on the new architecture and improvements, [read Daniel's readme](https://github.com/Ergodice/lczero-training/blob/attention-net-body/README.md)).
+
+#### Monte Carlo Graph Search
+In 2020, Crazyara's author and two others released a [paper](https://arxiv.org/abs/2012.11045) describing a way of handling transpositions. Transpositions are a big problem in AlphaZero-like search -- it's just wasting visits and computation!
```

Comment:
* `a way of handling transpositions` → `improvements to the Monte-Carlo search that handles transpositions better` maybe?
* `Transpositions are a...` → `Currently, when several variations end up in the same positions in chess, AlphaZero-like search evaluates them several time, wasting visits and computation.`